### PR TITLE
[Api] Receiving percentage discount

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

This PR is based on https://github.com/Sylius/Sylius/pull/12081

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Shop/CartContext.php [hunk 1]
```diff
@@ -541,4 +565,22 @@ private function checkProductQuantity(
             }
         }
     }
+
+    private function compareItemSubtotal(ProductInterface $product, \Closure $price): void
```
Comment (arti0090) [inline src/Sylius/Behat/Context/Api/Shop/CartContext.php:569]: I see that very simillar or even the same function is here https://github.com/Sylius/Sylius/pull/12081/files/a2cef239265f11c566a0efbff8514ef2142685d4#diff-b5c163d690bfe47026cba0b16b71bdc7a4e1b0d95352b10522d4b0194d23e85dR555
Maybe we can think about another class that can have same function? 😉

**Review by arti0090 — APPROVED** [review]

**Review by AdamKasp — APPROVED** [review]

File: src/Sylius/Behat/Context/Api/Shop/CartContext.php (hunk 1 above)
Comment (Tomanhez) [reply src/Sylius/Behat/Context/Api/Shop/CartContext.php:569]: This is the same function because this PR is based on #12081

**Review by GSadee — COMMENTED** [review]
Could you rebase with master?

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Tomasz! :tada:

**Pull request merged**